pkg/urbit/jets/e/secp.c

```diff
@@ -207,7 +207,7 @@ u3qe_reco(u3_atom has,
   ret = secp256k1_ecdsa_recover(ctx_u,                      /* IN: context */
                                 & puk_u,                     /* OUT: pub key */
                                 & sig_u,                        /* IN: signature */
-                                (const c3_y *) & has);        /* IN: message has */
+                                has_y);                       /* IN: message has */
```

Comment:
That's brutal. Looks like it would work for direct atoms, which is probably how it got past smoke-testing.